core/types/taiko_transaction.go

```diff
@@ -9,7 +9,7 @@ func (tx *Transaction) IsAnchor() bool {
 }
 
 func (tx *DynamicFeeTx) isAnchor() bool {
-	return tx.isAnhcor
+	return tx.anchor
 }
 
 func (tx *LegacyTx) isAnchor() bool {
```

Comment:
return tx.anchor?

Reply:
Yeah, we should avoid conflict between the variable name and the function name.